source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,94 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+``JsonObject`` is a more performant alternative to both ``Document``
+and ``BsonDocument`` when working with JSON strings because it does not create an intermediate
+``Map`` object.
+
+By default, ``JsonObject`` stores :manual:`Extended JSON </reference/mongodb-extended-json/>`.
+You can customize the format of JSON in ``JsonObject`` by specifying a
+:java-docs:`JsonObjectCodec</apidocs/bson/org/bson/codecs/JsonObjectCodec.html>`
+and passing it a
+:java-docs:`JsonWriterSettings</apidocs/bson/org/bson/json/JsonWriterSettings.html>`
+object. For more information on JSON formats, see
+our :doc:`Extended JSON guide </fundamentals/data-formats/document-data-format-extended-json/>`.
+For more information on specifying codecs, see our 
+:doc:`Codecs guide </fundamentals/data-formats/codecs>`.
+
+In the following code snippet, we show how to instantiate a sample ``JsonObject`` 
+instance wrapping an Extended JSON string containing different types of key value pairs:
+
+.. code-block:: java
+
+   String ejsonStr =  "{"
+                     +" \"_id\": {"
+                     +"  \"$oid\": \"6035210f35bd203721c3eab8\""
+                     +" },"
+                     +" \"name\": \"Gabriel Garc\\u00eda M\\u00e1rquez\","
+                     +" \"dateOfDeath\": {"
+                     +"  \"$date\": \"2014-04-17T04:00:00Z\""
+                     +" },"
+                     +" \"novels\": ["
+                     +"  {"
+                     +"   \"title\": \"One Hundred Years of Solitude\","
+                     +"   \"yearPublished\": 1967"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Chronicle of a Death Foretold\","
+                     +"   \"yearPublished\": 1981"
+                     +"  },"
+                     +"  {"
+                     +"   \"title\": \"Love in the Time of Cholera\","
+                     +"   \"yearPublished\": 1985"
+                     +"  }"
+                     +" ]"
+                     +"}";
+   JsonObject author = new JsonObject(ejsonStr);
+
+To insert this document into a collection, instantiate a collection using
+:java-docs:`getCollection()</apidocs/mongodb-driver-sync/com/mongodb/client/MongoDatabase.html#getCollection(java.lang.String,java.lang.Class)>`,
+specifying the ``JsonObject`` class. Then, call the
```

Comment:
Since there are more than one parameters, it's helpful to identify which one you're referring to.
```suggestion
specifying the ``JsonObject`` class as the document class parameter. Then, call the
```

Reply:
Thanks for the feedback. This language was inherited from the `BsonDocument` and `BasicDBObject` sections. I agree that mentioning the specific `getCollection` parameter name is more precise. The three instances of this language are changed in https://github.com/mongodb/docs-java/pull/57/commits/18fa5aa6a68b6b373c46094f70716616c653e457